src/exco/extractor/locator/built_in/below_of_locator.py

```diff
@@ -20,8 +21,8 @@ def locate(self, anchor_cell_location: CellLocation,
             for cell in row:
                 if cell.value == self.label:
                     cell_loc = CellLocation(
-                        sheet_name=anchor_cell_location.sheet_name,
-                        coordinate=util.shift_coord(cell.coordinate, (1, 0))
+                                sheet_name=anchor_cell_location.sheet_name,
```

Comment:
you will end up in an infinite loop here .... seriously, please get it right.

Reply:
is this even correct??????????? you should only check below the row but you check the entire row.